akka-actor-typed-tests/src/test/scala/akka/actor/typed/AskSpec.scala

```diff
@@ -129,7 +129,8 @@ class AskSpec extends ScalaTestWithActorTestKit("""
       val answer: Future[String] = actor.ask(Foo("bar", _))
       val result = answer.failed.futureValue
       result shouldBe a[TimeoutException]
-      result.getMessage should startWith("Ask timed out on")
+      // Message in the TimeoutException depends on the race between the actor stopping and
+      // the PromiseRef initializing... it's not really important for this test
```

Comment:
Could also check that exactly one of the patterns (`^Ask timed out on.*$` or `^Recipient.* had already terminated.$`) matches, if we want to test that this only times out in one of two ways, but not sure on the value there.

Reply:
Actually, the original purpose of this line is mock an actor dead, and expected both future failure and dead letters.
Therefore, the "ask timeout out" is an mistake on PR #32104.